docs/en/modules/gpio.md

```diff
@@ -0,0 +1,108 @@
+# GPIO Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2016-11-26 | [DiUS](https://github.com/DiUS), [Johny Mattsson](https://github.com/jmattsson) | [Johny Mattsson](https://github.com/jmattsson) | [gpio.c](../../../components/modules/gpio.c)|
```

Comment:
Actually, this one was done solely on my own time, DiUS doesn't get credit this time =)